### PR TITLE
documentation added

### PR DESCRIPTION
**Pull request by Haaroon**
added documentation for the algorithms. this mirrors the docs on raphtory.github.io

### CODE REVIEW

File: src/main/scala/com/raphtory/algorithms/LPA.scala [hunk 1]
```diff
@@ -11,19 +11,21 @@ Description
...
 Parameters
-  top (Int)       – The number of top largest communities to return. (default: 0)
-                      If not specified, Raphtory will return all detected communities.
-  weight (String) - Edge property (default: ""). To be specified in case of weighted graph.
-  maxIter (Int)   - Maximum iterations for LPA to run. (default: 500)
-  seed    (Long)  - The seed for randomness, if empty then random seed is selected by system
+  top (Int) : The number of top largest communities to return. (Default: 0) If not specified, Raphtory will return all detected communities.
+  weight (String) : Edge property (Default: ""). To be specified in case of weighted graph.
+  maxIter (Int) : Maximum iterations for algorithm to run. (Default: 500)
+  seed (Lond) : Value used for the random selection, can be set to ensure same result is returned per run. If not specified, it will generate a random seed.
```
Comment (miratepuffin) [inline src/main/scala/com/raphtory/algorithms/LPA.scala:17]: Long

File: src/main/scala/com/raphtory/algorithms/LPA.scala [hunk 2]
```diff
@@ -11,19 +11,21 @@ Description
...
-  communities (List(List(Long))) – Communities sorted by their sizes. Returns largest top communities if specified.
+  ID (Long) : Vertex ID
+  Label (Long) : The ID of the community this belongs to
 
 Notes
-  This implementation of LPA incorporated probabilistic elements which makes it non-deterministic;
-  The returned communities may differ on multiple executions.
-  **/
+  This implementation of LPA incorporated probabilistic elements which makes it
+  non-deterministic; The returned communities may differ on multiple executions.
```
Comment (miratepuffin) [inline src/main/scala/com/raphtory/algorithms/LPA.scala:26]: . or no capital

File: src/main/scala/com/raphtory/algorithms/MotifAlpha.scala [hunk 3]
```diff
@@ -2,12 +2,18 @@ package com.raphtory.algorithms
...
-/*
-
-This is an implementation of the alpha_1 motif from the paper
-Detecting Mixing Services via Mining Bitcoin Transaction Network With Hybrid Motifs
- */
-
+/**
+Description
+  The algorithms identifies 2-edge-1-node temporal motifs; It detects one type of motifs:
+  For each vertex, for each incoming edge it checks whether there are any outgoing edges that occur after
```
Comment (miratepuffin) [inline src/main/scala/com/raphtory/algorithms/MotifAlpha.scala:8]: For each incoming edge a vertex has, the algorithm checks whether there are any outgoing edges which occur after it and returns a count of these.

File: src/main/scala/com/raphtory/algorithms/TriangleCount.scala [hunk 4]
```diff
@@ -2,6 +2,38 @@ package com.raphtory.algorithms
 
 import com.raphtory.core.model.algorithm.{GraphAlgorithm, GraphPerspective, Row}
 
+/**
+Description
+  Returns the number of triangles and clustering coefficient.
```
Comment (miratepuffin) [inline src/main/scala/com/raphtory/algorithms/TriangleCount.scala:7]: Should we add clustering coefficient to the algorithm name?

**Review by miratepuffin — CHANGES_REQUESTED** [review]

File: src/main/scala/com/raphtory/algorithms/LPA.scala (hunk 1 above)
Comment (Haaroon) [reply src/main/scala/com/raphtory/algorithms/LPA.scala]: fixed

**Pull request merged**